draft-ietf-ppm-dap.md

```diff
@@ -1371,11 +1345,53 @@ following generic checks.
    the same. If so, then the Aggregator MUST mark the input share as invalid
    with error "unrecognized_message".
 
-1. Finally, if an Aggregator cannot determine if an input share is valid, it
-   MUST mark the input share as invalid with error `report_dropped`. This
-   situation arises if, for example, the Aggregator has evicted from long-term
-   storage the state required to perform the check. (See
-   {{reducing-storage-requirements}} for details.)
+1. If an Aggregator cannot determine if an input share is valid, it
+   MUST mark the input share as invalid with error `report_dropped`. For
+   example, if the Aggregator has evicted the state required to perform the
+   check from long-term storage. (See {{reducing-storage-requirements}} for
+   details.)
+
+1. Some VDAFs allow a batch to be collected multiple times; the aggregators must
+   check that the exact same set of reports is being aggregated each time.
+   A report may not be aggregated into a batch unless it was included in all
+   previous collections for the batch. (Checking that no batches are omitted
+   from a batch is covered in {{batch-validation}}.)
```

Comment:
```suggestion
   previous collections for the batch (checking that no reports are omitted
   from a batch is covered in {{batch-validation}}).
```

Reply:
These seem like two sentences to me so I'm not sure why we would remove the split. I'm removing the parentheses instead.